docs/management/managing-indices.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[managing-indices]]
+== Managing Indices
+
+include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+The index management UI enables you to update your Elasticsearch indices
+either individually or in bulk. To open the UI, go to *Management* and click
+*Index Management*.
+
+[role="screenshot"]
+image::images/management-index-management.png[Index Management UI]
+
+Clicking an index name opens a panel with access to the index settings, mapping, and
+statistics. Operations for managing the index are in a context menu on the lower
```

Comment:
In general, I prefer to avoid describing the behavior of the UI in too much detail. Here, I'd just say:

Click the name of an index to display the index summary and access the index settings, mappings, and statistics. The *Index options* menu on the lower right of the index pane enables you to manage the selected index. 

Are you using *on* for locations on the screen? (I don't feel strongly wrt in/on, just wondering if that's the guideline.)